app/subscriber/src/features/my-reports/edit/content/ContentEditForm.tsx

```diff
@@ -215,7 +268,13 @@ export const ContentEditForm = React.forwardRef<HTMLDivElement | null, IContentE
             </Col>
             <Col className="sentiment">
               <Sentiment
-                value={form.content.tonePools.length ? form.content.tonePools[0].value : undefined}
+                value={
+                  userFormTonePool
```

Comment:
This will result in always using their own tonepool.  The user needs to choose if they want to use their own tonepool.  In most cases users will not change the sentiment.

Reply:
created a toggle to change between user tone and editor tone.